packages/synapse-react-client/src/components/TableColumnSchemaEditor/ColumnModelFormFields/DefaultValueField.test.tsx

```diff
@@ -91,16 +105,24 @@ describe('DefaultValueField', () => {
         isOriginallyDefaultColumn: false,
       },
       onChange,
-      value: '2021-01-15T00:00:00.000Z',
+      value: '1710388800000',
```

Comment:
Bad test - the backend passes a string containing an integer unix timestamp with millisecond precision